omnigibson/robots/tiago.py

```diff
@@ -697,11 +698,45 @@ def arm_workspace_range(self):
             "right": [np.deg2rad(-75), np.deg2rad(-15)],
         }
 
-    def get_position_orientation(self):
-        # TODO: Investigate the need for this custom behavior.
-        return self.base_footprint_link.get_position_orientation()
+    def get_position_orientation(self, frame=RelativeFrame.WORLD):
+        """
+        Gets tiago's pose with respect to the specified frame.
+
+        Args:
+            frame (RelativeFrame): frame to get the pose with respect to. Default to WORLD. PARENT frame
+            get position relative to the object parent. SCENE frame get position relative to the scene.
+
+        Returns:
+            2-tuple:
+                - 3-array: (x,y,z) position in the specified frame
+                - 4-array: (x,y,z,w) quaternion orientation in the specified frame
+        """
+
+        if frame == RelativeFrame.WORLD:
+            return self.base_footprint_link.get_position_orientation()
+
+        elif frame == RelativeFrame.SCENE:
+
+            # TODO: Add the ability to get the position and orientation of the robot in the scene frame
+            pass
+        else:
+
+            # TODO: Add the ability to get the position and orientation of the robot in the parent frame
+            pass
```

Comment:
All three options here should simply be delegated to the base_footprint_link.get_position_orientation